src/platform_impl/apple/appkit/window_delegate.rs

```diff
@@ -881,7 +882,7 @@ impl WindowDelegate {
         // state, since otherwise we'll briefly see the window at normal size
         // before it transitions.
         if attrs.visible {
-            if attrs.active {
+            if attrs.active && attrs.focusable {
```

Comment:
Nit: Could you add a comment about why this is desirable? Something to the effect of "if the window isn't focusable, we don't want it to take keyboard focus on creation either".